### PR TITLE
[DE-425] Implementes client for doppler legacy

### PR DESCRIPTION
**Pull request by Cromeror**

### CODE REVIEW

File: src/implementations/DopplerLegacyClientImpl.ts [hunk 1]
```diff
@@ -0,0 +1,50 @@
+import {
+  DopplerLegacyClient,
+  DopplerLegacyUserData,
+} from "../abstractions/doppler-legacy-client";
+import { ResultWithoutExpectedErrors } from "../abstractions/common/result-types";
+import { AxiosResponse, AxiosStatic } from "axios";
+import { AppConfiguration } from "../abstractions";
+
+export class DopplerLegacyClientImpl implements DopplerLegacyClient {
+  private instance;
```
Comment (andresmoschini) [inline src/implementations/DopplerLegacyClientImpl.ts:10]: What do you think about naming it `axios`?

```suggestion
  private axios: Axios;
```

File: src/implementations/DopplerLegacyClientImpl.ts [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        await this.instance.get("/WebApp/GetUserData");
+      const { jwtToken, unlayerUser } = axiosResponse.data;
+      return {
+        success: true,
+        value: {
+          jwtToken,
+          unlayerUser: {
+            id: unlayerUser.id,
+            signature: unlayerUser.signature,
+            email: unlayerUser.email,
```
Comment (andresmoschini) [inline src/implementations/DopplerLegacyClientImpl.ts:39]: I have removed email field: https://github.com/FromDoppler/doppler-editors-webapp/pull/46

```suggestion
```

Comment (andresmoschini) [reply src/implementations/DopplerLegacyClientImpl.ts:39]: See https://github.com/FromDoppler/doppler-editors-webapp/blob/22a29fc6343ded129084a662c02b099c030e479f/src/abstractions/doppler-legacy-client/index.ts#L3-L6

File: src/default-configuration.ts [hunk 3]
```diff
@@ -10,4 +10,5 @@ export const defaultAppConfiguration: AppConfiguration = {
     "https://cdn.fromdoppler.com/unlayer-editor/asset-manifest-v2.json",
   loaderUrl: "https://cdn.fromdoppler.com/loader/v1/loader.js",
   unlayerProjectId: 32092,
+  dopplerLegacyBaseUrl: "https://appint.fromdoppler.net",
```
Comment (andresmoschini) [inline src/default-configuration.ts:13]: Remember to create a new PR in Swarm repository with the configuration for each environment, you should update the [HTML template](https://github.com/MakingSense/doppler-swarm/blob/master/webapps-stack/editors/index.template.html) and the environment files

File: src/implementations/DopplerLegacyClientImpl.ts [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+    axiosStatic: AxiosStatic;
+    appConfiguration: Partial<AppConfiguration>;
+  }) {
+    this.instance = axiosStatic.create({
+      baseURL: dopplerLegacyBaseUrl,
+      withCredentials: true,
+    });
+  }
+
+  async getDopplerUserData(): Promise<
```
Comment (andresmoschini) [inline src/implementations/DopplerLegacyClientImpl.ts:25]: What do you think about adding a couple of tests for this function?

You can mock Axios.

File: src/default-configuration.ts (hunk 3 above)
Comment (Cromeror) [reply src/default-configuration.ts:13]: Its Ok, https://github.com/MakingSense/doppler-swarm/blob/0019fee3689b17c4e24f8d9a93a4bc7abb4e5054/webapps-stack/editors/index.template.html#L60

File: src/implementations/DopplerLegacyClientImpl.ts (hunk 2 above)
Comment (Cromeror) [reply src/implementations/DopplerLegacyClientImpl.ts:39]: Yep, I fixed that.

File: src/implementations/DopplerLegacyClientImpl.ts (hunk 1 above)
Comment (Cromeror) [reply src/implementations/DopplerLegacyClientImpl.ts:10]: I will change 👍

File: src/default-configuration.ts (hunk 3 above)
Comment (andresmoschini) [reply src/default-configuration.ts:13]: Oh, I forgot it hehe, thanks!

**Pull request merged**